config/crds/virt_v1alpha1_storagemap.yaml

```diff
@@ -0,0 +1,81 @@
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  creationTimestamp: null
+  labels:
+    controller-tools.k8s.io: "1.0"
+  name: storagemaps.virt.konveyor.io
+spec:
+  group: virt.konveyor.io
+  names:
+    kind: StorageMap
+    plural: storagemaps
+  scope: Namespaced
+  subresources:
+    status: {}
+  validation:
+    openAPIV3Schema:
+      properties:
+        apiVersion:
+          description: 'APIVersion defines the versioned schema of this representation
+            of an object. Servers should convert recognized schemas to the latest
+            internal value, and may reject unrecognized values. More info: https://git.k8s.io/community/contributors/devel/api-conventions.md#resources'
+          type: string
+        kind:
+          description: 'Kind is a string value representing the REST resource this
+            object represents. Servers may infer this from the endpoint the client
+            submits requests to. Cannot be updated. In CamelCase. More info: https://git.k8s.io/community/contributors/devel/api-conventions.md#types-kinds'
+          type: string
+        metadata:
+          type: object
+        spec:
+          properties:
+            map:
+              description: Map.
+              items:
+                properties:
+                  destination:
+                    description: Destination storage.
+                    properties:
+                      storageClass:
+                        description: A storage class.
+                        type: string
+                    required:
+                    - storageClass
+                    type: object
+                  source:
+                    description: Source storage.
+                    properties:
+                      id:
+                        description: 'The network identifier. For:   - vsphere: The
```

Comment:
Network should be storage

Reply:
Strange, I don't think this matches the common on the struct.

Reply:
Likely due to anonymous struct.  Will refactor with named struct.  This lends itself to the _source_ being reused for both network and storage so will make the comment more generic.
Thinking the anonymous struct may not be a good idea.  Also, found that the field being an anonymous struct make is hard (impossible) to fix the unit tests.